DISCLAIMER.md

```diff
@@ -0,0 +1,21 @@
+# Disclaimer
+
+Simple Icons asks that users read this disclaimer fully before including an icon in their project.
+
+## Table of contents
+
+* [Copyright & Trademark](#copyright--trademark)
+* [Guidelines](#guidelines)
+* [Removal of Brands](#removal-of-brands)
+
+## Copyright & Trademark
+
+The Simple Icons package contains a number of logos with specific requirements around their use. Where this is the case, we aim to provide access to the proper guidelines through the 'guidelines' property in the NPM package, or through links on our website. We ask that our users read these terms fully, and display logos at their own risk. Simple Icons cannot be held responsible for any legal activity raised by a brand. If the icon includes a (registered) trademark icon we follow the guidelines [in our Contribution Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#icon-guidelines) to decide whether to include the symbol or not.
```

Comment:
I'm not sure about the last line of this section (hence the strikethrough); it's probably something we should seek some sort of legal advice on. Either way, it wouldn't do any harm to run this document past someone with legal experience once we finalise it.

```suggestion
The Simple Icons library contains a number of icons with specific requirements around their usage. Where this is the case, we aim to provide access to the proper guidelines through the `guidelines` entry in our JSON file & in our NPM package and through links on our [website](http://simpleicons.org/).

We ask that users of our library read these guidelines fully and display the icons in accordance with them and, where applicable, with the necessary permissions.

If a logo or icon includes a registered trademark or trademark symbol we follow the recommendations outline [in our Contribution Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#icon-guidelines) to decide whether to include the symbol or not.

~Simple Icons cannot be held responsible for any legal activity raised by a brand over our users' usage of their icon.~
```